homeassistant/components/modbus/climate.py

```diff
@@ -301,6 +305,20 @@ def __init__(
         else:
             self._hvac_onoff_coil = None
 
+        if CONF_CURRENT_TEMP_SCALE in config and CONF_CURRENT_TEMP_OFFSET in config:
+            self._current_temp_scale = config[CONF_CURRENT_TEMP_SCALE]
+            self._current_temp_offset = config[CONF_CURRENT_TEMP_OFFSET]
+        else:
+            self._current_temp_scale = None
+            self._current_temp_offset = None
+
+        if CONF_TARGET_TEMP_SCALE in config and CONF_TARGET_TEMP_OFFSET in config:
```

Comment:
This looks wrong, you should be able to use scale without offset.

And please use .get as I remarked earlier.

Reply:
I don't understand. In lines 174, 205, 233, 269, etc., .get is not used, but instead it is written in the style of “in config”. Why should I have it in a different format?

Reply:
I get the distinct feeling you do not want my review comments ! Feel free to ask for a review by someone else.

I am getting into a "why" discussion, that is NOT the purpose of a review.

Reply:
I just thought everything should be in the same style. As usual in “style guides.”